src/harness/fake_sas.py

```diff
@@ -307,6 +313,8 @@ def do_POST(self):
       response = FakeSas().Relinquishment(request)
     elif self.path == '/%s/deregistration' % self.version:
       response = FakeSas().Deregistration(request)
+    elif self.path == '/admin/injectdata/ex_zone':
```

Comment:
Please use the URL given in the testing API doc.

Reply:
Corrected the URL

Reply:
This does not appear to be done.